webapp/components/NewsItem.vue

```diff
@@ -2,43 +2,22 @@
   <div>
     <h2>{{ news.title }}({{ news.votes }})</h2>
     <button @click="upvote" v-if="isAuthenticated">Upvote</button>
-    <button @click="downvote" v-if="isAuthenticated">Downvote</button>
-    <button @click="remove" v-if="isAuthenticated">Remove</button>
+    <button v-if="isAuthenticated">Downvote</button>
+    <button v-if="isAuthenticated">Remove</button>
```

Comment:
isAuthenticated ist here für "remove" nicht genügend glaube ich. Also "remove" ist nur für Author

Reply:
genau, das wird in "buttons based on state" realisiert